opm/grid/common/ZoltanGraphFunctions.hpp

```diff
@@ -135,7 +135,7 @@ class CombinedGridWellGraph
                           const std::vector<OpmWellType> * wells,
                           const double* transmissibilities,
                           bool pretendEmptyGrid,
-			  EdgeWeightMethod edgeWeightsMethod);
+                          EdgeWeightMethod edgeWeightsMethod);
```

Comment:
tabs should be removed

Reply:
That is a removal of tabs. With this PR there are no tabs in ZoltanGraphFunctions.hpp. If you do: git grep -P "\t", you see where the tabs are.